### ISSUE TITLE
BaasBox has to log the original exception when it detects that a transaction has been left open due an non-catched exception

### ISSUE THREAD

**Issue by giastfader** [baasbox/baasbox#698]

**Comment by giastfader** [comment]
should be enough to change the method DbHelper.withDbFromContext():

```Java
public static F.Function0<play.mvc.Result> withDbFromContext(Http.Context ctx, F.Function0<Result> work){
		return ()->{
			try{
				DbHelper.openFromContext(ctx);
				return work.apply();
			}catch (Throwable e){
				if (isInTransaction()) DbHelper.rollbackTransaction();
				throw e;
			}finally {
				DbHelper.close(DbHelper.getConnection());
			}
		};
	}
```

**Comment by giastfader** [comment]
closed by #812
